File: Android Client-server apps/ClientServerWithVolleySample/app/src/main/java/ru/guar7387/clientserverwithvolleysample/utils/Logger.java
package ru.guar7387.clientserverwithvolleysample.utils;

import android.util.Log;

public class Logger {

    private Logger() { }

    private static final boolean isDebug = true;

    public static void log(String tag, String message) {
        if (isDebug) {
            Log.i(tag, message);
        }
    }

}

File: Android Kotlin/KotlinFunny/app/src/main/kotlin/ru/guar7387/kotlinfunny/basics/Logger.kt
package ru.guar7387.kotlinfunny.basics

import android.util.Log
import ru.guar7387.kotlinfunny

public class Logger {

    private val isDebug = true;

    class object {
        fun log(tag: String, message: String) {
            if (isDebug) {
                Log.i(tag, message)
            }
        }
    }

}



